### PR TITLE
prep for Quietust's structure updates

### PR DESCRIPTION
**Pull request by myk002**
more fields have disapperated

@RosaryMala If you have work for this already prepped, we can merge yours and I can revert this one.

### CODE REVIEW

File: library/Core.cpp [hunk 1]
```diff
@@ -2266,6 +2266,7 @@ bool Core::ncurses_wgetch(int in, int & out)
     }
     if(in >= KEY_F(1) && in <= KEY_F(8))
     {
+/* TODO: understand how this changes for v50
```
Comment (lethosor) [inline library/Core.cpp:2269]: What part of this is problematic? As far as I can tell, hotkeys are about the same.

Comment (myk002) [reply library/Core.cpp:2269]: the reference to `ui`. It's just a name change, but the new names aren't actually in the tree yet.

Comment (myk002) [reply library/Core.cpp:2269]: and `ui_sidebar_mode`

**Comment by RosaryMala** [comment]
"Work" is a strong word for what I did. Just commented out things until it compiled with no added notes or anything.
I'll be reverting that and pulling this.

**Pull request merged**